boost/boost.bzl

```diff
@@ -164,11 +118,10 @@ def boost_deps():
         strip_prefix = "boost-1.84.0",
     )
 
-    # We're pointing at hedronvision's mirror of google/boringssl:master-with-bazel to get Renovate auto-update. Otherwise, Renovate will keep moving us back to master, which doesn't support Bazel. See https://github.com/renovatebot/renovate/discussions/24854
-    maybe(
-        http_archive,
-        name = "openssl",
-        url = "https://github.com/hedronvision/boringssl/archive/0f1a639954dd7ab86f5f4ddd8b4e2edbea492acd.tar.gz",
-        sha256 = "7ce152bdce1b85344cc36c6b255aab36905d39187c2c2f797a69d5ad220076ee",
-        strip_prefix = "boringssl-0f1a639954dd7ab86f5f4ddd8b4e2edbea492acd",
+    http_archive(
+        name = "zlib",
```

Comment:
Why do you keep zlib here? It is available in the bazel registry so you could get it via bzlmod

Reply:
See the above error in this PR! If this is merged, then an issue should be created to resolve this.

Reply:
https://github.com/nelhage/rules_boost/pull/565

See here for the currently failing build when we switch zlib to BCR